langages/scala/programmer-en-scala-3-booleens-et-conditions.md

```diff
@@ -1,4 +1,10 @@
-# Programmer en Scala 3 : Booléens et conditions
+---
+layout: post
+author: Raphaël Fromentin et TheElectronWill
+date: 2022-09-16
```

Comment:
```suggestion
date: 2023-03-10
```

Reply:
https://github.com/NotANameServer/Not-a-Hub/commits/master/langages/scala/programmer-en-scala-3-booleens-et-conditions.md